_layouts/widget-map.html

```diff
@@ -0,0 +1,156 @@
+---
+layout: documentation-single
+---
+
+<link id="widget-theme-simple" rel="stylesheet" href="{{ "/products-and-docs/widgets/map/1.0.0/theme/simple.css" | prepend: site.baseurl }}">
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-slider/6.1.5/css/bootstrap-slider.min.css">
+
+<div class="widget-content-wrapper">
+	<div class="row row-native">
+		<div class="col-lg-12 col-md-12 col-sm-12">
+			<h1>MAP WIDGET</h1>
+			<p class="lead">The TM Widgets allow partners to setup particular widgets and provide enhanced discovery experience for consumers within copy/paste distance.</p>
+			<h2>Widget: Keyword search</h2>
+			<p>You can use the widget configurator below to customize the layout of the widget, and grab a small code snippet to insert into your website.</p>
+		</div>
+	</div>
+
+	<div class="row row-native">
+		<div class="col-lg-4 config-block">
+			{% include widget-map-config.html %}
+		</div>
+
+		<div class="col-lg-8 col-lg-widget">
+			<div class="widget-container-wrapper">
+				<div class="widget-container widget-container--map">
+					<div
+						w-type="map"
+						w-tmapikey=""
+						w-googleapikey=""
+						w-keyword=""
+						w-theme="simple"
+						w-colorscheme="light"
+						w-width="300"
+						w-height="560"
+						w-size="25"
+						w-border="0"
+						w-borderradius="4"
+						w-postalcode="90015"
```

Comment:
I propose, Implement from start w-latlong as a main parameter. The widget should not make extra request to Google to get latlon from a zipcode.